tests/test_02_04_model.py

```diff
@@ -0,0 +1,162 @@
+import os
+from datetime import datetime
+from unittest.case import TestCase
+
+from bs4 import BeautifulSoup
+
+# ignoring E402: module level import not at top of file
+os.environ['TMA_CERTIFICATE'] = 'cert content'  # noqa: E402
+os.environ['BRP_APPLICATIE'] = 'mijnAmsTestApp'  # noqa: E402
+os.environ['BRP_GEBRUIKER'] = 'mijnAmsTestUser'  # noqa: E402
+os.environ['MKS_BRP_ENDPOINT'] = 'https://example.com'  # noqa: E402
+
+from mks.model.stuf_02_04 import extract_data
+
+FIXTURE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'fixtures')
+RESPONSE_PATH = os.path.join(FIXTURE_PATH, "response_0204.xml")
+
+
+class Model0204Tests(TestCase):
+    def get_result(self):
+        return {
+            'adres': {
+                'adresInOnderzoek': False,
+                'begindatumVerblijf': datetime(2009, 1, 1, 0, 0),
+                'centroidXCoordinaat': None,
```

Comment:
Ah, komen deze ook mee? Dan hoef ik niet meer naar BAG

Reply:
Als dat zo is, kun je dan `centroid: [centroidXCoordinaat, centroidYCoordinaat]` toevoegen?

Reply:
Nee, ze komen niet mee, meer wishful thinking. Wilde kijken wat het deed op prod.